### ISSUE TITLE
Launch trouble

### ISSUE THREAD

**Issue by LoneSimba** [maiorBoltach/bf2142stats_emu#3]
Hello, i have some trouble with launching fesl_login_server. On start, even as Administartor, is have error 0xc000007b. OpenSSL is installed, webserver is Open Server 5.2.8. I copied libmysql.dll to fesl_login_server folder. Any help?

**Comment by LoneSimba** [comment]
i used Win32 and Win64 OpenSSL from [here](http://bvsoft.ru/download/utils/openssl/)

**Comment by maiorBoltach** [comment]
Hi, @LoneSimba . I have started fesl on WinServer 2016, [Win32OpenSSL-1.0.2o](https://slproweb.com/products/Win32OpenSSL.html) and [MySql Connector C 6.1.11 win32](https://dev.mysql.com/downloads/connector/c/) (i haven't copied libmysql from openserver). Please, try this configuration

**Comment by maiorBoltach** [comment]
ToDo: Fix install instruction (to up-to-date)

**Comment by GhostNaix** [comment]
I Believe I have figured out the issue.
fesl_login_server **does not work with libmysql.dll** that is **x64** 
You must find a libmysql.dll that is **x86**

**TLDR:** Fesl_login_Server does not work with the libmysql.dll shipped in the latest version of MySQL
